kbase-extension/static/kbase/templates/data_staging/ftp_file_table.html

```diff
@@ -1,41 +1,46 @@
-<table id="kb-data-staging-table" class="table table-striped table-condensed" style="table-layout:fixed">
-    <thead>
+<table class="kb-staging-table table table-striped table-condensed">
+    <thead class="kb-staging-table-header">
         <tr>
-            <th style="width: 5%"></th>
-            <th style="width: 40%; overflow:hidden">Name</th>
-            <th style="width: 13%">Size</th>
-            <th style="width: 12%">Age</th>
-            <th style="width: 30%">Import As...</th>
+            <!-- TODO: what behavior should select all checkbox have? -->
+            <th class="kb-staging-table-header__checkbox">
+                <input type="checkbox" name="select_all" value="1" id="">
+            </th>
+            <th class="kb-staging-table-header__file"></th>
+            <th class="kb-staging-table-header__name" scope="col">Name</th>
+            <th class="kb-staging-table-header__size" scope="col">Size</th>
+            <th class="kb-staging-table-header__age" scope="col">Age</th>
+            <th class="kb-staging-table-header__import" scope="col">Import As...</th>
         </tr>
     </thead>
-    <tbody>
+    <tbody class="kb-staging-table-body">
     {{#each files}}
         <tr>
-            <td style="text-align:center">{{isFolder}}</td>
+            <td>{{name}}</td>
+            <td class="kb-staging-table-body__expander">{{isFolder}}</td>
             <td>{{name}}</td>
             <td>{{size}}</td>
             <td>{{mtime}}</td>
-            <td>
+            <td class="kb-staging-table-body__import">
                 {{#if imported }}
-                <div class="kb-data-mgmt">
+                <div>
                     <select>
                         <option></option>
                         {{#each ../uploaders}}
                             <option value="{{id}}">{{name}}</option>
                         {{/each}}
                     </select>
-                    <button data-import="{{name}}" class="btn btn-xs btn-primary">
+                    <!-- <button data-import="{{name}}" class="btn btn-xs btn-primary">
                         <span><i class="fa fa-upload"></i></span>
-                    </button>
-                    <button data-download="{{name}}" class="btn btn-xs btn-primary">
+                    </button> -->
+                    <button data-download="{{name}}" class="kb-staging-table-body__download btn btn-xs btn-primary">
```

Comment:
If the button doesn't have any text, it is useful to add an aria-label to inform screen readers what the button does. Same for the following buttons. https://developer.mozilla.org/en-US/docs/Learn/Accessibility/WAI-ARIA_basics